docker/Makefile.docker

```diff
@@ -0,0 +1,39 @@
+
+XDEBUG_REMOTE_HOST := $(shell hostname -I | grep -o "^\S*")
```

Comment:
This is the point, which needs extra testing on windows in order for XDebug to work.